iota/commands/extended/is_promotable.py

```diff
@@ -0,0 +1,40 @@
+# coding=utf-8
+from __future__ import absolute_import, division, print_function, \
+  unicode_literals
+
+import filters as f
+from iota import TransactionHash
+from iota.commands import FilterCommand, RequestFilter
+from iota.commands.core.check_consistency import CheckConsistencyCommand
+from iota.filters import Trytes
+
+__all__ = [
+  'IsPromotable',
+]
+
+
+class IsPromotableCommand(FilterCommand):
+  """
+  Executes ``isPromotable`` extended API command.
+
+  See :py:meth:`iota.api.Iota.is_promotable` for more info.
+  """
+  command = 'isPromotable'
+
+  def get_request_filter(self):
+    return GetBundlesRequestFilter()
```

Comment:
Should this be `IsPromotableRequestFilter`?

Reply:
Yes. Fixed. I'm sure the CI would have eventually caught that with a `NameError` but you beat it to it